cli/tests/containerize.bats

```diff
@@ -131,17 +113,18 @@ function skip_if_linux() {
   run "$FLOX_BIN" containerize
   assert_success
 
-  assert [ -f "test-container.tar.gz" ] # <env-name>-container.tar.gz by default
+  assert [ -f "test-container.tar" ] # <env-name>-container.tar by default
 
   run which podman
 
-  run podman load -i test-container.tar.gz
+  run podman load -i test-container.tar
   assert_success
   assert_line --partial "Loaded image:"
 }
 
 # bats test_tags=containerize:piped-to-stdout
 @test "catalog: container is written to stdout when '-o -' is passed" {
+  skip "duplicate of next test"
```

Comment:
Can we just drop this test instead?

Reply:
I suspect we will add it back in soon with variations in the upcoming sprints. No strong-opinion either way.